archinstall/lib/user_interaction.py

```diff
@@ -259,16 +276,16 @@ def get_keyboard_input(self, strip_rowbreaks=True, end='\n'):
 
 		poller.register(sys.stdin.fileno(), select.EPOLLIN)
 
-		EOF = False
-		while EOF is False:
+		eof = False
+		while eof is False:
```

Comment:
`EOF` is a known term, more accurately it's an abbreviation and those are spelled out in capital letters AFAIK.

Reply:
variables in local functions are supposed to be lowercase